samples/web/content/apprtc/apprtc.py

```diff
@@ -10,6 +10,7 @@
 import cgi
 import client
 import constants
+import decline_page
```

Comment:
Will it be better to group the local module together, separated from the built-in/third_party modules?